src/Environment.php

```diff
@@ -5,6 +5,8 @@
 class_exists('Twig_Environment');
 
 if (\false) {
+    @trigger_error(sprintf('Using the "Twig\Environment" class is deprecated since Twig version 1.38, use "Twig\Twig" instead.'), E_USER_DEPRECATED);
+
     class Environment extends \Twig_Environment
```

Comment:
What about adding `@deprecated`, so IDEs can highlight this class is deprecated?